src/cli/mod.rs

```diff
@@ -26,8 +26,14 @@ pub(crate) struct FlakeHubPushCli {
         default_value = "https://api.flakehub.com"
     )]
     pub(crate) host: url::Url,
-    #[clap(long, env = "FLAKEHUB_PUSH_VISIBLITY")]
+
+    #[clap(long, env = "FLAKEHUB_PUSH_VISIBILITY")]
     pub(crate) visibility: crate::Visibility,
```

Comment:
I think these both have to be `Option`s, otherwise we're still requiring the corrected spelling for consumers.

Reply:
I've made these both `Option`s but not sure if the `match` statement is necessary. Let me know if not.

Reply:
The `match` looks good.